python/hail/utils/misc.py

```diff
@@ -38,6 +38,7 @@ def new_temp_file(n_char=10, prefix=None, suffix=None):
                             'MEMORY_AND_DISK_SER_2', 'OFF_HEAP')
 
 _test_dir = None
+_doctest_dir = None
```

Comment:
This doesn't look used... what is it for?

Reply:
I'm pulling a file from `python/docs/data` for the python tests. Had it for the docs and didn't want to put it in multiple places.

Reply:
Ah I see what's going on here.